admin/tabs/wizard.php

```diff
@@ -119,5 +95,60 @@ public function right_column( $setting ) {
   <!-- End Box -->
         <?php
     }
+    public function load_options( $options, $step ) {
+        ?>
+      <form onsubmit="onClickOptionButton(event)">
+      <table class="widefat striped">
+            <thead>
+              <tr>
+                <th colspan="3"><?php echo 'Set Options' ?></th>
+              </tr>
+            </thead>
+            <tbody>
+            <?php
+            foreach ( $options as $option ){
+                $key = $option->key;
+                $value = $option->value;
+                $db_value = get_option( $key );
+                if ( gettype( $value ) == 'object' ){
+                    $value = json_encode( $value );
+                }
+                if ( gettype( $db_value ) == 'object' ){
+                    $db_value = json_encode( $db_value );
+                }
+                ?>
+              <tr>
+                <td>
+                <?php echo esc_html( $key ); ?>
+                </td>
+                <td>
+                <?php echo esc_html( $value ); ?>
+                </td>
+                <td>
+                <?php
+                if ( $value == $db_value ){
+                    echo 'Done!';
+                } else {
+                    ?>
+                  <input type="text" name=<?php echo esc_html( $key ) ?> value="<?php echo esc_html( $value ) ?>" />
+                  <button type="submit" name="button" value="<?php echo esc_html( $key )  ?>">
```

Comment:
Can the button be in a separate cell from the form input? It sometimes wraps below and is a bit awkward then.